File: CS502017/pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
    {
        int h;
        do
        {
            printf("Height:");
            h = get_int();
        }
        while (h<0||h>23);
        for (int i = 0; i<h; i++)
        {
           for(int s = h-2; s>=i; s--)
           {
               printf(" ");
           }
           for(int d = -2; d<i; d++)
           {
               printf("#");
           }
           for(int p = h; p==h; p++)
           {
               printf("\n");
           }
        }
    
    }

File: CS502018/pset6/sentimental/mario/mario.py
#imports the cs50 library
import cs50

#initalize the input variable
input = 0

#keeps looping until user enters correct input
while True:
    input = cs50.get_int("Height: ")

    if input >= 0 and input <= 23:
        break

#initalize space and block variable
space = input - 1
block = 1

#loops through the rows
for i in range(input):

    #creates the space before the blocks
    for j in range(space):
        print(" ", end="")

    #creates the block
    for k in range(block):
        print("#", end="")

    #prints the gap
    print("  ", end="")

    #prints the second set of blocks
    for k in range(block):
        print("#", end="")

    #prints the space
    print()

    #decrement the space and block
    space -= 1
    block += 1